src/index-debug.html

```diff
@@ -26,6 +26,9 @@
     <meta charset="utf-8">
     <title>Photoshop Design Space (development build)</title>
     <link rel="stylesheet" type="text/css" href="style.css" />
+    <script type="text/javascript">
+        var loadTime = new Date();
```

Comment:
I guess all declarations at the top-level end up in the global scope. Weird. Anyway, I don't like the idea of polluting the global scope. Can we do this locally in `main.startup`?